### PR TITLE
Oren/int c e

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by assafshp** [comment]
jenkins-test

File: src/components/examUtility/services/subScore.service.js [hunk 1]
```diff
@@ -1,87 +0,0 @@
-(function(angular){
```
Comment (assafshp) [inline src/components/examUtility/services/subScore.service.js]: not needed?

File: src/components/performance/services/performanceData.service.js [hunk 2]
```diff
@@ -67,13 +72,15 @@
...
+                    var mathAndVerbalPerformanceData = {
+                        '0': mathSubjectPerformanceData,
+                        '7': verbalSubjectPerformanceData
+                    };
 
-                    var mathAndVerbalSubScoreData = {};
-                        mathAndVerbalSubScoreData [SubjectEnum.MATH.enum] = {};
-                        mathAndVerbalSubScoreData [SubjectEnum.VERBAL.enum] = {};
+                    var mathAndVerbalSubScoreData = {
+                        '0': {},
```
Comment (assafshp) [inline src/components/performance/services/performanceData.service.js]: 0 and 7 should be enums?

File: src/components/examUtility/services/subScore.service.js (hunk 1 above)
Comment (orenor10) [inline src/components/examUtility/services/subScore.service.js]: need this service. it was duplicate

File: src/components/performance/services/performanceData.service.js (hunk 2 above)
Comment (orenor10) [inline src/components/performance/services/performanceData.service.js]: fixed

**Pull request merged**